configurator/migrations/Version20250415153255.php

```diff
@@ -0,0 +1,111 @@
+    }
+    
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+      
+        $this->keycloakManager->putRealm([
+            'displayName'               => 'Phrasea Auth',
+            'displayNameHtml'           => '<div class="kc-logo-text"><span>Phrasea Auth</span></div>',
+            'registrationAllowed'       => $this->getBooleanEnv('KC_REALM_LOGIN_REGISTRATION_ALLOWED', false),
+            'resetPasswordAllowed'      => $this->getBooleanEnv('KC_REALM_LOGIN_RESET_PASSWORD_ALLOWED', true),
+            'rememberMe'                => $this->getBooleanEnv('KC_REALM_LOGIN_REMEMBER_ME_ALLOWED', true),
+            'loginWithEmailAllowed'     => $this->getBooleanEnv('KC_REALM_LOGIN_WITH_EMAIL_ALLOWED', true),
+            'verifyEmail'               => $this->getBooleanEnv('KC_REALM_LOGIN_VERIFY_EMAIL_ALLOWED', false),
+            'registrationEmailAsUsername' => $this->getBooleanEnv('KC_REALM_LOGIN_EMAIL_AS_USERNAME', false),
+            'editUsernameAllowed'       => $this->getBooleanEnv('KC_REALM_LOGIN_EDIT_USERNAME', false),            
+             'bruteForceProtected'       => true,
+            'failureFactor'             => '30',   
+            'bruteForceStrategy'        => 'MULTIPLE',
+            'permanentLockout'          => false,
+            'waitIncrementSeconds'      => '60',
+            'maxFailureWaitSeconds'     => '900',
+            'maxDeltaTimeSeconds'       => '43200',
+            'quickLoginCheckMilliSeconds' => '1000',
+            'minimumQuickLoginWaitSeconds' => '60',
+            'eventsEnabled'                 => $this->getBooleanEnv('KC_REALM_USER_EVENT_ENABLED', false),
+            'eventsExpiration'              => getenv('KC_REALM_USER_EVENT_EXPIRATION') ?: '604800',
+            'eventsListeners'               => ['jboss-logging'],
+            'adminEventsEnabled'           => $this->getBooleanEnv('KC_REALM_ADMIN_EVENT_ENABLED', false),
+            'adminEventsDetailsEnabled'    => true,      
+            'ssoSessionIdleTimeout'     => getenv('KC_REALM_SSO_SESSION_IDLE_TIMEOUT') ?: '1800',
+            'clientSessionIdleTimeout'  => getenv('KC_REALM_CLIENT_SESSION_IDLE_TIMEOUT') ?: '1800',
+            'offlineSessionIdleTimeout' => getenv('KC_REALM_OFFLINE_SESSION_IDLE_TIMEOUT') ?: '2592000',
+            'internationalizationEnabled' => true,
+            'supportedLocales'          => (getenv('KC_REALM_SUPPORTED_LOCALES') != null) ? explode(',', getenv('KC_REALM_SUPPORTED_LOCALES')) : ['en'],
+            'defaultLocale'             => getenv('KC_REALM_DEFAULT_LOCALE') ?: 'en',
+            'smtpServer' => [
+                'auth' => getenv('MAILER_USER') ? true : false,
+                'from' => getenv('MAIL_FROM') ?: 'noreply@phrasea.io',
+                'fromDisplayName' => getenv('MAIL_FROM_DISPLAY_NAME') ?: 'Phrasea',         
+                'replyTo' => getenv('MAIL_REPLAY_TO') ?: '',
```

Comment:
typo on MAIL_REPLY_TO ?
please add 
“Reply to display name” 
and  
“enveloppe from”